### PR TITLE
Drupal.js

### PR DESCRIPTION
**Pull request by DFurnes**
ok this is actually the opposite of what we're doing but come on just look at it this is great

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/js/config.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
     baseUrl: "/profiles/dosomething/themes/dosomething/paraneue_dosomething/js/",
     include: "main",
     paths: {
-      "jquery": "../bower_components/jquery/dist/jquery",
+      "jquery": "../bower_components/jquery/jquery",
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/js/config.js]: Why is this path changing?

**Comment by aaronschachter** [comment]
This all looks good technically, but are can you think of any gotchas we need to look out for, to make sure this doesn't break something unintentionally?

**Comment by weerd** [comment]
Boo. Looks fine, but yeah, agree with @aaronschachter Keep in mind certain syntax things... like .on() and stuff...

**Comment by weerd** [comment]
Just checked and `.on()` was added as of v1.7 so that should be fine, but we should still double check stuff @_@

**Comment by weerd** [comment]
also @DFurnes :+1: on the image ;)

File: lib/themes/dosomething/paraneue_dosomething/js/config.js (hunk 1 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/js/config.js]: The jQuery folks changed the path things were bundled up in 1.11.x. So we're movin' back to the old ways.

**Comment by DFurnes** [comment]
Double-checked the 1.9-1.11 release notes and the major changes are just performance improvements and removing deprecated APIs (which is the gotcha that's causing Drupal to hurt, although admittedly the `$.browser` one that it's been crying about has been deprecated since v1.3 c.a. January 2009 so...)

**Comment by aaronschachter** [comment]
not sure what the `$.browser` thing means but hey lets rock it :+1:

**Comment by DFurnes** [comment]
This should be good to merge. Also ended up cleaning up some of the RequireJS configuration so net win. 

:chart_with_upwards_trend:

**Comment by weerd** [comment]
Cleanup looks cool :+1: :sun_with_face:

**Pull request merged**